crates/re_types_builder/src/codegen/common.rs

```diff
@@ -45,6 +41,44 @@ pub fn get_documentation(docs: &Docs, tags: &[&str]) -> Vec<String> {
     lines
 }
 
+pub fn get_examples(
+    docs: &Docs,
+    extension: &str,
+    prefix: &[&str],
+    suffix: &[&str],
+) -> anyhow::Result<Vec<String>> {
+    let mut lines = Vec::new();
+
+    if let Some(examples) = docs.tagged_docs.get("example") {
+        let base_path = crate::rerun_workspace_path().join("docs/code-examples");
+
+        let mut examples = examples.iter().peekable();
+        while let Some(example) = examples.next() {
+            let example = example.trim();
+            let path = base_path.join(format!("{example}.{extension}"));
+            let contents = std::fs::read_to_string(&path)
+                .with_context(|| format!("couldn't open code example {path:?}"))?;
```

Comment:
In wonder if we should really require that all languages has all examples. Some examples only makes sense in some languages, e.g. an example of how to convert a python PIL image to a Rerun image, or similar.

I think it makes sense to require that each language has at least one example per archetype though.

Reply:
I was going to add stricter checks (check that each archetype has an `\example`), but decided to hold off for now. I think this would be good to discuss during standup